integrations/spring-boot/src/it/core-config/src/main/resources/application.properties

```diff
@@ -7,4 +7,6 @@ one.microstream.data-file-minimum-size=1MB
 one.microstream.data-file-maximum-size=2MB
 one.microstream.data-file-minimum-use-ratio=0.9
 one.microstream.data-file-cleanup-head-file=true
-one.microstream.entity-cache-timeout=1d
\ No newline at end of file
+one.microstream.entity-cache-timeout=1d
+
+one.microstream.auto-start=false
```

Comment:
one.microstream.storage.auto-start
will be better. Microstream contains more features than storage only.